.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Description of the change**
+
+Clearly and concisely describe the purpose of the pull request. If this PR relates to an existing issue or change proposal, please link to it. Include any other background context that would help reviewers understand the motivation for this PR.
+
+---
+
+**Checklist:**
+
+- [ ] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
+- [ ] Linked any existing issues or proposals that this pull request should close
+- [ ] Updated or added relevant documentation in the README and/or documentation directory
```

Comment:
Perhaps we should remove "in the README and/or documentation directory" for core libraries and just leave the documentation note.

Reply:
Sounds good to me.